app/routes/posts/show.js

```diff
@@ -4,6 +4,11 @@ export default Ember.Route.extend({
   model(params) {
     return this.get('store').find('post', params.post_id);
   },
+  emptyTextArea: Ember.on('activate', function(){
+    setTimeout(function(){
+      $(".ember-text-area").val("").focus();
+    },10)
+  }),
```

Comment:
This is emptying the comments text area on transitioning away from the show page. I have a setTimeout because if you type something inside the comment box and navigate to another page and then come back the text will still be in the text area.